org.eclipse.jdt.ls.core/src/org/eclipse/jdt/ls/core/internal/cleanup/INoConfigCleanUp.java

```diff
@@ -0,0 +1,43 @@
+/*******************************************************************************
+ * Copyright (c) 2022 Red Hat Inc. and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License 2.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *     Red Hat Inc. - initial API and implementation
+ *******************************************************************************/
+package org.eclipse.jdt.ls.core.internal.cleanup;
+
+import org.eclipse.core.runtime.CoreException;
+import org.eclipse.jdt.core.manipulation.CleanUpContextCore;
+import org.eclipse.jdt.core.manipulation.ICleanUpFixCore;
+
+/**
+ * Represents a cleanup change that doesn't need further configuration and
+ * cannot support further configuration.
+ *
+ * @author datho7561
+ */
+public interface INoConfigCleanUp {
```

Comment:
Any strong preference for `INoConfigCleanUp` over something like `ISimpleCleanUp` ? If you like it more feel free to switch. If not, I guess we can keep this. I would just add in the javadoc above something like _doesn't need further configuration (eg. UI interaction)_

Reply:
I like `ISimpleCleanUp` better, so I changed it to that.